subsys/mgmt/mcumgr/mgmt/src/mgmt.c

```diff
@@ -106,9 +106,12 @@ int32_t mgmt_callback_notify(uint32_t event, void *data, size_t data_size)
 		struct mgmt_callback *loop_group =
 			CONTAINER_OF(snp, struct mgmt_callback, node);
 
-		if (loop_group->event_id == event || loop_group->event_id == MGMT_EVT_OP_ALL ||
+		if (loop_group->event_id == MGMT_EVT_OP_ALL ||
 		    (MGMT_EVT_GET_GROUP(loop_group->event_id) == group &&
-		     MGMT_EVT_GET_ID(loop_group->event_id) == MGMT_EVT_OP_ID_ALL)) {
+		     (MGMT_EVT_GET_ID(loop_group->event_id) == MGMT_EVT_OP_ID_ALL ||
```

Comment:
This line is probably no longer needed now as with `MGMT_EVT_OP_ID_ALL = 0xffff` it reduces to the second condition below.

Reply:
Fixes.